sdrf_pipelines/openms/unimod.py

```diff
@@ -35,6 +41,15 @@ def get_name(self):
     def get_accession(self):
         return self._ontology_term.get_accession()
 
+    def get_delta_mono_mass(self):
```

Comment:
Another option would be to make them properties with the "@property" decorator. That way the "get_" prefix could be left out and the attributes would still have some protection against overwriting.